rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1033,6 +1038,24 @@ public long getFeePerKb(Object[] args)
         return bridgeSupport.getFeePerKb().getValue();
     }
 
+    public long getLockingCap(Object[] args) {
+        logger.trace("getLockingCap");
+
+        return bridgeSupport.getLockingCap().getValue();
+    }
+
+    public boolean increaseLockingCap(Object[] args) {
+        logger.trace("increaseLockingCap");
+
+        Coin newLockingCap = BridgeUtils.getCoinFromBigInteger((BigInteger)args[0]);
```

Comment:
It would be nice to see a test where we check that the overflow check is happening, in case by some of those laws which seem to govern software, the condition is forgotten, removed or changed by someone in the future :)